src/views/about/about.astro

```diff
@@ -1,71 +1,64 @@
-import { Picture } from "@astrojs/image/components";
 import ContributorCard from "./contributor-card.astro";
 import { getTranslatedPage, translate } from "src/utils/translations";
-import { Button } from "components/index";
-import { Icon } from "astro-icon";
-import TranslationsHeader from "../base/translations/translations-header.astro";
+import TranslationsHeader from "src/views/base/translations/translations-header.astro";
+import License from "./license.astro";
+import { siteMetadata } from "constants/site-config";
+import BannerSvg from "./banner-svg.astro";
 
 const about = getTranslatedPage(
 	Astro,
-	await Astro.glob("../../../content/site/about-us*.md"),
-);
-const sponsors = getTranslatedPage(
-	Astro,
-	await Astro.glob("../../../content/site/sponsors*.md"),
+	await Astro.glob("../../../content/site/about-us*.mdx"),
 );
 ---
 
 <div class="container">
-	<div class="d-flex text-center">
-		<Picture
-			src={import("../../assets/unicorn_utterances_logo_512.png")}
-			loading={"eager"}
-			widths={[192, 56]}
-			sizes={"192px"}
-			formats={["avif", "webp", "png"]}
-			alt={"Unicorn Utterances logo"}
-			class={style.unicornLogo}
-		/>
-		<h1 class="text-style-headline-1">{translate(Astro, "title.about_us")}</h1>
+	<div class={style.banner}>
+		<BannerSvg />
 	</div>
 	<main class={`${style.aboutBody} post-body`}>
 		<TranslationsHeader locales={about.locales} />
+
 		<about.page.Content />
 
-		<div class={`${style.links} d-flex text-center`}>
-			{
-				Object.entries(data.about.links).map(([name, link]) => (
-					<Button variant="primary-emphasized" href={link.url}>
-						<Icon slot="leftIcon" width="100%" height="100%" name={link.icon} />
-						{name}
-					</Button>
-				))
-			}
-		</div>
+		<hr />
 
-		<h2>{translate(Astro, "title.sponsors")}</h2>
-		<div class={`${style.sponsors} d-flex text-center`}>
+		<h2 id="contributors">
+			{translate(Astro, "title.contributors")}
+		</h2>
+		<div class={style.unicorns}>
 			{
-				data.about.sponsors.map((sponsor) => (
-					// these are all svgs, so astro's Picture/Image components make no sense here
-					<a href={sponsor.url} class={style.sponsor}>
-						<img src={sponsor.logo} width="300" alt={sponsor.name} />
-					</a>
+				data.unicorns.map((unicornInfo) => (
+					<ContributorCard unicorn={unicornInfo} />
 				))
 			}
 		</div>
-		<sponsors.page.Content />
 
-		<h2>{translate(Astro, "title.contributors")}</h2>
-		<div class={style.unicorns}>
+		<hr />
+
+		<h2 id="licenses">{translate(Astro, "title.licenses")}</h2>
+
+		<div class={style.licenses}>
 			{
-				data.unicorns.map((unicornInfo) => (
-					<ContributorCard unicorn={unicornInfo} />
-				))
+				Object.values(data.tags).map((tag) =>
+					"image" in tag ? (
+						<License
+							name={tag.displayName}
+							image={tag.image}
+							url={
+								`https://github.com/${siteMetadata.repoPath}/blob/main/public` +
+								tag.image.replace(".svg", "-LICENSE.txt")
+							}
```

Comment:
Instead of showing all `tag`s, I think we should just show the licenses for the stickers, alongside the usage of `explainer`:

https://github.com/unicorn-utterances/unicorn-utterances/blob/uwu/src/utils/get-stickers.ts#L16-L17